AntistasiOfficial.Altis/CREATE/createSupplyBox.sqf

```diff
@@ -111,42 +138,41 @@ spawner setVariable [_marker, 0, true]; //Activate when merged with new spawn sy
 	_counter: running timer
 */
 
-//waitUntil{sleep 1; ([300,1 ,_crate,"BLUFORSpawn"] call distanceUnits) OR ({_x distance2D _crate < 1000} count puestosFIA != 0)};
 //Reveal marker
 _marker setMarkerAlpha 1;
 if([300,1,_crate, "BLUFORSpawn"] call distanceUnits) then
 {
 	//Crate detected by player
 	{
-			["You detected an AAF supply crate near you!"] remoteExec ["hint",_x];
+		["You detected an AAF supply crate near you!"] remoteExec ["hint",_x];
 	} forEach ([300,0,_crate,"BLUFORSpawn"] call distanceUnits);
 }
 else
 {
 	//Crate detected by watchpost
 	[[petros,"globalChat","Our watchposts have detected an AAF supply crate. Check your maps!"],"commsMP"] call BIS_fnc_MP;
 };
+// 150
+[false, false, 5, 0] params ["_isCrateUnloaded", "_timerRunning", "_deploymentTime", "_counter"];
 
-[false,150,0] params ["_timerRunning","_deploymentTime","_counter"];
-//TODO rework marker
-while {alive _crate AND (_marker in markerSupplyCrates)} do {
+while {alive _crate OR (_marker in markerSupplyCrates)} do {
+    sleep 1;
 
-	// wait until the player loads the crate or have the loaded crate in a city
-	waitUntil {sleep 5;
-		!(isNull attachedTo _crate) OR
-		!({(_crate distance (getmarkerpos _x) < 200)} count ciudades == 0) OR
-		!(_marker in markerSupplyCrates)
+	// wait until the player loads the crate
+	waitUntil {
+        sleep 1;
+		!(isNull attachedTo _crate)
 	};
```

Comment:
Maybe add an "_marker setMarkerAlpha 0" here, cause the marker will stay visible on the loadpoint, may causing players to search a crate which isn´t there any more.
Maybe change to usage of inmuneConvoy.sqf

Reply:
ok then just moove `deleteMarker _marker;` here from l.253

Reply:
Maybe the combination of` _marker setMarkerAlpha 0; spawner setVariable [_marker, nil, true];` will do a better job. Needs to be tested